modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/select.d.ts

```diff
@@ -58,5 +58,6 @@ interface SelectProps
 
 type MultiSelectItem = {
 	label: string;
+	reference: any;
```

Comment:
Hey @cgmonte, is there any reason to use any here?

Reply:
No, that was a mistake. Thanks for catching this. I will fix and send an update.